src/metatensor/models/cli/train_model.py

```diff
@@ -302,8 +306,8 @@ def _train_model_hydra(options: DictConfig) -> None:
     OmegaConf.save(config=options, f=Path(output_dir) / "options.yaml")
 
     logger.info("Setting up model")
-    architetcure_name = options["architecture"]["name"]
-    architecture = importlib.import_module(f"metatensor.models.{architetcure_name}")
+    architecture_name = options["architecture"]["name"]
```

Comment:
Suggestion: shall we add a check of the model's architecture before attempting to import it? Like if the architecture is in the list of available architectures. Otherwise, it may just fail on import and say that the model is not available, however it might be just a typo in the architecture name.

Reply:
Yes! We have a whole function that does it thanks to @PicoCentauri 
https://github.com/lab-cosmo/metatensor-models/blob/c08d3fd1a9ac06d9f5b5d780c6945c254c74aa23/src/metatensor/models/cli/train_model.py#L83
It checks whether there might be a typo and suggests the closest architecture. It also suggests experimental/deprecated architectures if the architecture is not available as a stable architecture

Reply:
Amazing!